src/api/http.ts

```diff
@@ -1,11 +1,17 @@
+/* eslint-disable immutable/no-mutation */
 import { Store } from 'redux'
-import axios, { Axios } from 'axios-observable'
+import Axios from 'axios-observable'
 import { AxiosRequestConfig } from 'axios'
 
-import { authInterceptor } from 'api/interceptors/auth'
+import { authInterceptor } from 'api/interceptors/auth.interceptor'
 
 import { RootState } from 'state/store'
 
+const baseURL = process.env.REACT_APP_API_URL
+  ? `https://${process.env.REACT_APP_API_URL}`
```

Comment:
why not just use process.env.REACT_APP_API_URL? 
and set REACT_APP_API_URL to https://prod-url on prod ????

Reply:
I have tryed and it now worked